### PR TITLE
Init development of cli

### PR DESCRIPTION
**Pull request by mehdivk**
cc @rprieto @nguyenchr @stamf

Initial development of cli.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+Deletes left over channels (channels without subscriber) in given topic.
+
+##Installation
+```
+npm install -g nsq-cleanup
+```
+
+##Usage
+`nsq-cleanup` accepts following parameters:
+ - `-u, --url`: Url to the NSQD HTTP API.
```
Comment (rprieto) [inline README.md]: the code says `-h` :)

**Comment by rprieto** [comment]
Looks great! :+1: 

Is there a way to specify `channels that haven't been read from in X minutes`?
(e.g. in case the subscriber went offline for a bit, just as the CRON job ran)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nsq-cleanup",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/.bin/mocha test"
+  },
+  "author": "Tabcorp Digital Technology Team",
```
Comment (nguyenchr) [inline package.json:9]: don't forget to add the repository and issues field if this is a public repo

Comment (mehdivk) [inline package.json:9]: @nguyenchr Added

File: README.md (hunk 1 above)
Comment (mehdivk) [inline README.md]: @rprieto last minute refactoring :), fixed it

**Comment by nguyenchr** [comment]
:+1:

**Pull request merged**